webapp-django/crashstats/base/templates/crashstats_base.html

```diff
@@ -144,26 +144,26 @@
         <div class="login">
             <ul>
                 <li>
-                    {% if request.user.is_authenticated() %}
-                      {{ request.user.email }}
-                      |
-                      <a href="{{ url('crashstats:your_crashes') }}">Your Reports</a>
-                      |
-                      <a href="{{ url('crashstats:permissions') }}">Permissions</a>
-                      |
-                      <a href="{{ url('tokens:home') }}">API Tokens</a>
-                      |
-                      <a href="{{ url('symbols:home') }}">Symbols</a>
-                      {% if request.user.is_superuser %}
-                      |
-                      <a href="{{ url('manage:home') }}">Admin</a>
-                      {% endif %}
-                      |
-                      {{ browserid_logout(text='Sign out') }}
-                    {% else %}
-                      {# the redirect_next function comes from the browserid context processor #}
-                      {{ browserid_login(text='Sign in', color='dark', next=redirect_next()) }}
+                {% if request.user.is_authenticated() %}
+                    <a href="{{ url('profile:profile') }}"
+                      title="Your Profile"
+                    >
+                        {{ request.user.email }}
```

Comment:
I don't like that. It makes it look like `<a href="mailto:me@dot.com">me@dot.com</a>`. I know that's not true but that's what my eyes see. 
I think you should reverse the words. So it's `<a href=... title="You are signed in as {{ request.user.email }}">Your Profile</a>`